app/models/manageiq/providers/embedded_ansible/automation_manager/credential.rb

```diff
@@ -6,9 +6,6 @@ class ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Credential < Mana
   alias_attribute :manager_id, :resource_id
   alias_attribute :manager, :resource
 
-  include ManageIQ::Providers::AnsibleTower::Shared::AutomationManager::Credential
```

Comment:
Yeah, as a part of the credentials piece of this it might be easiest to re-model the credentials we care about as non-inventory bits. Even if we shim the existing models so that we don't have to sort out all the callers and users.

I'm not sure if this is exactly what you were thinking here, but it would definitely get us away from sharing things with the tower implementation.

Reply:
As discussed on the call, I would be ok with denormalizing this, creating a complete separate copy, as it's likely the directions of both embedded ansible and external tower will diverge.

Reply:
Yup, agreed.  Sounds good.

I am considering also doing some follow up in the tower repo just to move all of the "shared" code to be just inline, to reduce the complexity of that repo now that it doesn't need it.

Reply:
That's definitely a future cleanup task, but lowest priority.

Reply:
@NickLaMuro So you *knew* about the UI using this but removed it anyway? :) Braino I guess ;)

https://github.com/ManageIQ/manageiq-ui-classic/issues/4921#issuecomment-501202974